envoy/config/filter/http/transcoder/v2/transcoder.proto

```diff
@@ -55,4 +55,16 @@ message GrpcJsonTranscoder {
   // `JsonPrintOptions <https://developers.google.com/protocol-buffers/docs/reference/cpp/
   // google.protobuf.util.json_util#JsonPrintOptions>`_.
   PrintOptions print_options = 3;
+
+
+  // Whether to skip recalculating the route after the
+  // outgoing headers have been transformed to the match the upstream gRPC service
+  // If set to true, Envoy will determine the cluster to
+  // route to based on the route for the incoming request
+  // Note: this will require routes to be added to match
+  // the incoming HTTP requests rather than the outgoing
+  // requests (after transcoding). This means routes that
+  // point directly to gRPC services cannot be reused for
```

Comment:
Please clarify what is meant here. I think you're saying that if a route in a `RouteConfiguration` is used for a gRPC service that is not transcoded, we can't make use of it in combination with `match_incoming_request_route` to perform transcoding. Is this correct?